src/Modal/Modal.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import classes from "./Modal.module.css";
+
+const Modal = props => {
+  return (
+    <div className={classes.modal}>
```

Comment:
Consider using the [dialog](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog) element instead of `div` for this component down the road!